dashboard/src/shared/url.ts

```diff
@@ -42,6 +42,8 @@ export const api = {
       `/api/kube/api/v1/namespaces/kubeapps/configmaps?labelSelector=NAME in (${releaseNames.join(
         ",",
       )})`,
+    upgrade: (namespace = "default", releaseName: string) =>
```

Comment:
Just FYI, we stopped using this and switched to classes to handle API requests, see the classes under `shared/` for examples. That said, more of an FYI as we can go back and update this in a future PR.